MapboxSceneKit/Routes/TerrainNodeExtension+Routes.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  TerrainNodeExtension+Routes.swift
+//  MapboxSceneKit
+//
+//  Created by Jim Martin on 8/16/18.
+//  Copyright © 2018 MapBox. All rights reserved.
+//
+
+import Foundation
+import SceneKit
+import CoreLocation
+
+extension TerrainNode {
+    
+    @objc
+    public func addPolyline( coordinates: [CLLocation], startRadius: CGFloat, endRadius: CGFloat, startColor: UIColor, endColor: UIColor) -> PolylineNode{
+        
+        var scenePositions : [SCNVector3] = []
+        for coord in coordinates {
+            let position = self.positionForLocation(coord)
+            scenePositions.append(position)
+        }
+        
+        //TODO: check along lines between coordinates for intersections with terrain geometry, and add points to avoid those interasections.
```

Comment:
Remove TODO or DO it

Reply:
I don't agree with this in this case. It's useful to stub out future effort in-line when it's an obvious extension that's just not in the scope of an individual PR, and also helps others think about where new contributions might fit.

Reply:
I agree that this TODO is valuable to keep. It makes a point that other devs could seize on to make improvements. Perhaps the way to handle it here is to create a ticket for this work and reference it here to make sure it is captured somewhere.